lib/fs.js

```diff
@@ -190,7 +193,7 @@ async function _copyDir(src, dest, options, parent) {
       return _copyDir(childSrc, childDest, options, currentPath);
     }
 
-    return copyFile(childSrc, childDest, options).thenReturn(currentPath);
+    return copyFile(childSrc, childDest).thenReturn(currentPath);
```

Comment:
The `options` here is for `copyDir` (`{ ignoreHidden, ignorePattern }`) which is not related with `copyFile`, so I just simply remove it.